treeherder/etl/job_loader.py

```diff
@@ -300,4 +300,4 @@ def _get_validated_jobs_by_project(self, jobs_list):
         return validated_jobs
 
     def _to_timestamp(self, datestr):
-        return time.mktime(parser.parse(datestr).timetuple())
+        return calendar.timegm(parser.parse(datestr).utctimetuple())
```

Comment:
TIL: In Python 2.7 there isn't a nicer way to perform this conversion, however in Python 3.3+ there's:
https://docs.python.org/3.3/library/datetime.html#datetime.datetime.timestamp

Side note: we really should switch Django to using UTC (bug 1209474).